packages/core/src/components/ExportSuccessModal.tsx

```diff
@@ -30,17 +30,129 @@ import { FiCheckCircle } from 'react-icons/fi'
+  const innerTabs = ['CSS / SCSS', 'Javascript / Typescript', 'Tailwind']
   return (
     <>
       <Text css={{ marginBottom: '24px' }}>
         Your tokens can now be imported by your app.
       </Text>
       <Tabs>
         <TabList>
-          <Tab>CSS / SCSS</Tab>
-          <Tab>Javascript / Typescript</Tab>
-          <Tab>Tailwind</Tab>
+          {outerTabs.map((tab) => {
+            return <Tab key={tab}>{tab}</Tab>
+          })}
+          {/* <Tab>Colors</Tab>
+          <Tab>Typography</Tab>
+          <Tab>Shadows</Tab> */}
+        </TabList>
+        <TabPanels>
+          <ColorsTab
+            tabs={innerTabs}
+            primaryName={
+              tokens && tokens.colorData[0]
+                ? tokens.colorData[0].name
+                : 'primary'
+            }
+            cssName="color"
+            cssPropertyName="background-color"
+            javascriptName="colors"
+            javascriptPropertyName="backgroundColor"
+            tailwindName="colors"
+            tailwindPropertyName="colors"
+          />
+          <TypographyTab
+            tabs={innerTabs}
+            primaryName={
+              tokens && tokens.typography.fontSizes[0]
+                ? tokens.typography.fontSizes[0].name
+                : 'sm'
+            }
+            cssName="font-size"
+            cssPropertyName="font-size"
+            javascriptName="fontSizes"
+            javascriptPropertyName="fontSize"
+            tailwindName="fontSizes"
+            tailwindPropertyName="fontSize"
+          />
+          <ShadowTab
+            tabs={innerTabs}
+            primaryName={
+              tokens && tokens.shadows[0] ? tokens.shadows[0].name : 'sm'
+            }
+            cssName="box-shadow"
+            cssPropertyName="box-shadow"
+            javascriptName="boxShadows"
+            javascriptPropertyName="boxShadow"
+            tailwindName="boxShadows"
+            tailwindPropertyName="dropShadow"
+          />
+        </TabPanels>
+      </Tabs>
+
+      <Text css={{ marginTop: '12px' }}>
+        To learn more about how to import these generated files, visit our{' '}
+        <Link
+          isExternal
+          color="blue.500"
+          href="https://mirrorful.com/docs/home/export-formats"
+        >
+          documentation here.
+        </Link>
+      </Text>
+      <Text css={{ marginTop: '8px' }}>
+        For examples, check out our{' '}
+        <Link
+          isExternal
+          color="blue.500"
+          href="https://mirrorful.com/docs/home/examples"
+        >
+          examples here.
+        </Link>
+      </Text>
+    </>
+  )
+}
+
+function ColorsTab(props: displayTabsProps) {
+  return <DisplayTabs {...props}></DisplayTabs>
+}
+
+function TypographyTab(props: displayTabsProps) {
+  return <DisplayTabs {...props}></DisplayTabs>
+}
+
+function ShadowTab(props: displayTabsProps) {
+  return <DisplayTabs {...props}></DisplayTabs>
+}
```

Comment:
These are all the same component/code, so I think for sake of simplicity we can just have one and generalize the name as  function `TokenTab` ?